<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SensorCalibrationWidget</class>
 <widget class="QWidget" name="SensorCalibrationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>393</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>171</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_camera_roll">
    <item>
     <widget class="QLabel" name="label_camera_roll">
      <property name="text">
       <string>Camera Roll</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_camera_roll">
      <property name="decimals">
       <number>3</number>
      </property>
      <property name="minimum">
       <double>-3.200000000000000</double>
      </property>
      <property name="maximum">
       <double>3.200000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.001000000000000</double>
      </property>
      <property name="value">
       <double>0.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>171</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_camera_yaw">
    <item>
     <widget class="QLabel" name="label_camera_yaw">
      <property name="text">
       <string>Camera Yaw</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_camera_yaw">
      <property name="decimals">
       <number>3</number>
      </property>
      <property name="minimum">
       <double>-3.200000000000000</double>
      </property>
      <property name="maximum">
       <double>3.200000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.001000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_body_roll">
    <item>
     <widget class="QLabel" name="label_body_roll">
      <property name="text">
       <string>Body Roll</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_body_roll">
      <property name="decimals">
       <number>3</number>
      </property>
      <property name="minimum">
       <double>-3.200000000000000</double>
      </property>
      <property name="maximum">
       <double>3.200000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.001000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>151</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_body_pitch">
    <item>
     <widget class="QLabel" name="label_body_pitch">
      <property name="text">
       <string>Body Pitch</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_body_pitch">
      <property name="decimals">
       <number>3</number>
      </property>
      <property name="minimum">
       <double>-3.200000000000000</double>
      </property>
      <property name="maximum">
       <double>3.200000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.001000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>241</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_field_x">
    <item>
     <widget class="QLabel" name="label_field_x">
      <property name="text">
       <string>Field Position X</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_field_x">
      <property name="minimum">
       <double>-300.000000000000000</double>
      </property>
      <property name="maximum">
       <double>300.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>241</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_field_y">
    <item>
     <widget class="QLabel" name="label_field_y">
      <property name="text">
       <string>Field Position Y</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_field_y">
      <property name="minimum">
       <double>-200.000000000000000</double>
      </property>
      <property name="maximum">
       <double>200.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>241</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_field_ori">
    <item>
     <widget class="QLabel" name="label_field_ori">
      <property name="text">
       <string>Field Pos. Orientation</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_field_ori">
      <property name="decimals">
       <number>3</number>
      </property>
      <property name="minimum">
       <double>-3.200000000000000</double>
      </property>
      <property name="maximum">
       <double>3.200000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.010000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>171</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_camera_pitch">
    <item>
     <widget class="QLabel" name="label_head_pitch">
      <property name="text">
       <string>Camera Pitch</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_camera_pitch">
      <property name="decimals">
       <number>3</number>
      </property>
      <property name="minimum">
       <double>-3.200000000000000</double>
      </property>
      <property name="maximum">
       <double>3.200000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.001000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Camera</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_camera_select">
      <item>
       <property name="text">
        <string>Darwin</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DarwinWebots</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="loadButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>350</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Save...</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textOutput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
